src/widgets/feedback-form/ui/FeedbackForm.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
+
+import { useState } from 'react';
+import { SolidButton } from '@/shared/ui/solid-button/SolidButton';
+import { usePostFeedback } from '@/features/feedback/model/usePostFeedback';
+import { TextArea } from '@/shared/ui/text-area/TextArea';
+
+export const FeedbackForm = () => {
+  const [content, setContent] = useState('');
+  const { mutate: submit, isPending } = usePostFeedback();
+
+  const handleSubmit = () => {
+    if (!content.trim() || isPending) return;
+    submit({ content });
+  };
```

Comment:
```suggestion
  const handleSubmit = () => {
    if (!content.trim() || isPending) return;
    submit(
      { content },
      {
        onSuccess: () => {
          setContent('');
        },
      },
    );
  };
```
피드백 보내기를 완료하면 mutate의 onSuccess 콜백을 이용하여 텍스트 상태를 초기화해주는 것은 어떨까요?